Resources/EditorData/AtomicEditor/editor/ui/buildsettings_android.tb.txt

```diff
@@ -20,40 +21,49 @@ TBLayout: axis: y, distribution: gravity, position: left
 		TBLayout: gravity: left right, distribution-position: right bottom
 			TBEditField: id: product_name, autofocus: 1
 				lp: min-width: 220
+	TBLayout: axis: y, position: left, distribution: gravity
+		TBLayout: gravity: left right, distribution-position: right bottom
+			TBEditField: id: icon_root, autofocus: 0
+				lp: min-width: 265
+			TBButton: id: choose_icon, lp: min-width: 48, min-height:48
+				tooltip location of the drawable folders for app icons
+				TBImageWidget: id: and_icon, filename: "Resources/ToolData/Deployment/Android/res/drawable-ldpi/icon.png"
 	TBSeparator: gravity: left right, skin: AESeparator
```

Comment:
The default TBImageWidget path should derived in code as a development build pulling assets from source tree, and release build pulling from packaged editor assets (possibly in an app bundle) may be different, I am a little fuzzy on this, though the default image isn't showing up on a packaged build, I can look into this... may be unrelated